SECURITY.md

```diff
@@ -0,0 +1,35 @@
+# Security
+
+Target takes the security of our software products and services seriously, which includes all source code repositories managed through [our GitHub organizations](https://opensource.target.com).
+
+If you believe you have found a security vulnerability in any Target-owned repository that meets Target's definition of a security vulnerability, please report it to us as described below.
+
+## Reporting Security Issues
+
+**Please do not report security vulnerabilities through public GitHub issues.**
+
+Instead, please report them to the Target Cyber Security team at _____.
+
+If you prefer to submit without logging in, send an email to _____. If possible, encrypt your message with our PGP key; please download it from _____.
+
+You should receive a response within 24 hours. If for some reason you do not, please follow up via email to ensure we received your original message. Additional information can be found at _____.
```

Comment:
thoughts on where more info be found? I think maybe saying something like 

`Additional information can be found if published to GitHub Advisories` for each repository` ??

Reply:
That seems reasonable - I don't think we'd have any other place to publish additional information for now (unless we want to create an org-level like what Microsoft has at -> https://www.microsoft.com/en-us/msrc?rtc=2 )

Reply:
Added